doc/source/cookbook.rst

```diff
@@ -1267,6 +1324,7 @@ The `method` argument within `DataFrame.corr` can accept a callable in addition
     ...     return cov_ab / std_a / std_b
     ...
     ...
+    ...
     >>> df = pd.DataFrame(np.random.normal(size=(100, 3)))
     ...
```

Comment:
doesn't seem like this line is required, is it?

Reply:
This df is reference in a subsequent line.

Reply:
I mean't the `...` in the next line, I don't think that is doing anything.

And I think the validation is failing because of the `...` you added before the `df = ...`. Can you remove that and see if that makes the CI green?

Reply:
Well here goes. One final try. Thanks for the guidance.